### ISSUE TITLE
Uncaught TypeError: Cannot read property 'firstChild' of undefined

### ISSUE THREAD

**Issue by revelt** [steelbrain/linter-ui-default#417]
I just open a file, in my case, with a `.nunjucks` extension (its type is detected as "HTML (Nunjucks Templates)" as shown in status bar). The error persists and obstructs the editor.

This is first time I'm using `1.20.0-beta2 x64`, it was not happening on previous version (I think before I was on `1.18.x` beta).

**Atom**: 1.20.0-beta2 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [linter-ui-default](https://github.com/steelbrain/linter-ui-default) package 1.6.5

### Stack Trace

Uncaught TypeError: Cannot read property 'firstChild' of undefined

```
At /Applications/Atom Beta.app/Contents/Resources/app/src/text-editor-component.js:2171

TypeError: Cannot read property 'firstChild' of undefined
    at TextEditorComponent.measureLongestLineWidth (/Applications/Atom Beta.app/Contents/Resources/app/src/text-editor-component.js:2171:112)
    at TextEditorComponent.measureContentDuringUpdateSync (/Applications/Atom Beta.app/Contents/Resources/app/src/text-editor-component.js:383:16)
    at TextEditorComponent.updateSync (/Applications/Atom Beta.app/Contents/Resources/app/src/text-editor-component.js:276:18)
    at TextEditorComponent.pixelPositionForScreenPosition (/Applications/Atom Beta.app/Contents/Resources/app/src/text-editor-component.js:202:18)
    at HTMLElement.pixelPositionForScreenPosition (/Applications/Atom Beta.app/Contents/Resources/app/src/text-editor-element.js:243:32)
    at getBufferPositionFromMouseEvent (/packages/linter-ui-default/lib/editor/helpers.js:12:47)
    at /packages/linter-ui-default/lib/editor/index.js:168:29)
    at later (/packages/linter/node_modules/sb-debounce/index.js:9:14)
```

### Commands

```
     -2:42.2.0 core:move-down (input.hidden-input)
     -2:41.9.0 core:move-up (input.hidden-input)
     -2:41.5.0 intentions:highlight (input.hidden-input)
     -2:41.4.0 editor:move-to-end-of-word (input.hidden-input)
  5x -2:41.1.0 core:move-right (input.hidden-input)
     -2:40.3.0 editor:select-to-end-of-word (input.hidden-input)
  3x -2:38.7.0 core:move-right (input.hidden-input)
     -2:38.2.0 core:select-right (input.hidden-input)
     -2:36.4.0 editor:consolidate-selections (input.hidden-input)
     -2:36.4.0 core:cancel (input.hidden-input)
     -2:36 editor:move-to-first-character-of-line (input.hidden-input)
     -2:31.8.0 core:move-down (input.hidden-input)
     -2:29 editor:delete-line (input.hidden-input)
     -2:28.3.0 core:move-left (input.hidden-input)
     -2:27.4.0 editor:move-to-first-character-of-line (input.hidden-input)
     -2:18.2.0 window:decrease-font-size (input.hidden-input)
```

### Non-Core Packages

```
ariake-dark-syntax 0.1.3 
atom-clock 0.1.13 
atom-easy-jsdoc 4.12.1 
atom-panic-palette-syntax 1.0.0 
atom-pug 0.2.0 
atom-react-snippets 0.2.2 
atom-wrap-in-tag 0.6.0 
aurora-theme 1.2.2 
auto-id-class 0.3.1 
auto-update-packages 1.0.1 
behave-theme 1.7.0 
browse 1.6.1 
busy-signal 1.4.3 
chester-atom-syntax 0.3.0 
close-tags 0.6.0 
color-town-syntax 0.1.4 
declarations 1.1.3 
declarations-js 1.0.2 
editorconfig 2.2.2 
file-icons 2.1.10 
flatwhite-syntax 0.8.1 
flow-ide 1.8.1 
gloom 0.6.4 
guess-indent 0.1.0 
hey-pane 1.0.0 
highlight-bad-chars 1.0.7 
hyperclick 0.1.3 
intentions 1.1.5 
jumpy 3.1.3 
language-babel 2.71.0 
language-diff 0.7.0 
language-markdown 0.25.1 
language-nunjucks 4.1.0 
language-pug 0.0.21 
linter 2.2.0 
linter-eslint 8.2.1 
linter-htmlhint 1.3.3 
linter-javac 1.9.4 
linter-js-standard 3.9.3 
linter-js-standard-engine 2.1.0 
linter-jsonlint 1.3.0 
linter-scss-lint 3.1.0 
linter-tidy 2.3.1 
linter-ui-default 1.6.5 
linter-write-good 0.9.0 
linter-xo 0.23.1 
markdown-preview-enhanced 0.14.7 
mixed-indent-warning 0.3.0 
modern-javascript-snippets 1.1.0 
monokai 0.24.0 
multi-cursor-increment 0.1.1 
no-title-bar 1.7.1 
nord-atom-syntax 0.9.1 
nord-atom-ui 0.11.0 
nucleus-dark-ui 0.12.3 
oceanic-next 1.0.0 
project-manager 3.3.5 
react 0.16.2 
sepia-syntax 1.1.0 
set-syntax 0.3.2 
simple-drag-drop-text 0.3.4 
split-diff 1.4.3 
standard-formatter 2.8.0 
tab-control 0.6.10 
text-manipulation 0.6.0 
webbox-color 0.5.9 
```

### Error popup

![error](https://user-images.githubusercontent.com/8344688/29309948-cf78ef5a-81a3-11e7-920b-f71026f005f3.png)

**Comment by steelbrain** [comment]
Error seems to be coming from Atom internal APIs, also you're on beta so this is not unexpected. You should check if an error with the similar stack has been reported to `atom/atom` repo, if not, open a new one.

Thanks!

**Comment by revelt** [comment]
Thanks for quick response. Will do!

**Comment by Arcanemagus** [comment]
Duplicate of https://github.com/atom/atom/issues/15311
